docs/ebpf-sqlite-io.md

```
@@ -0,0 +1,47 @@
+# SQLite I/O latency tracing (bpftrace)
+
+## Goal
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)